api/src/repositories/sample-location-repository.ts

```diff
@@ -0,0 +1,189 @@
+import { Feature } from 'geojson';
+import SQL from 'sql-template-strings';
+import { z } from 'zod';
+import { ApiExecuteSQLError } from '../errors/api-error';
+import { generateGeometryCollectionSQL } from '../utils/spatial-utils';
+import { BaseRepository } from './base-repository';
+import { PostSampleMethod } from './sample-method-repository';
+
+export interface PostSampleLocation {
```

Comment:
I wonder if we can move away from the `Put/Post` naming scheme and more towards `SomeObject` and `SomeObjectRecord`, since the latter is a bit more general

Reply:
Per discussion with Nick and Alfred, we're moving towards the `UpdateObject`, `InsertObject` and `ObjectRecord` naming scheme